etl/steps/data/grapher/oecd/2025-02-25/social_expenditure.py

```diff
@@ -0,0 +1,28 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("social_expenditure")
+
+    # Read table from garden dataset.
+    tb = ds_garden.read("social_expenditure", reset_index=False)
+
+    #
+    # Save outputs.
+    #
+    # Create a new grapher dataset with the same metadata as the garden dataset.
+    ds_grapher = create_dataset(
+        dest_dir, tables=[tb], check_variables_metadata=True, default_metadata=ds_garden.metadata
```

Comment:
if you want to import all tables from `ds_gaden` (I believe there is only one here), you can use `tables=list(ds_garden)`, without the need of reading any table explicitly.

Reply:
Right, thanks. I will change it here, though it's always easier to keep the default

Reply:
yeah, good point. no need to change